src/docs-ui/app/app.component.html

```diff
@@ -3,6 +3,38 @@
     <div td-toolbar-content layout="row" layout-align="start center" flex>
       <span flex></span>
 
+      <vui-account-profile name="donald duck" email="donald.duck@teradata.com">
```

Comment:
I would call this component `vui-account-menu` or `vui-profile-menu` as long as there is menu in the name.

Reply:
this component contains the button and the `vui-account-profile-menu` inside which can also menu used by itself

Reply:
Right, but when I read `vui-account-profile` I think a profile page. Might just be me though.